corpus/segments.py

```diff
@@ -261,6 +262,33 @@ def run(self):
                 f.writelines(lines[start:end])
 
 
+class DynamicSplitSegmentFileJob(Job):
+
+    def __init__(self, segment_file, concurrent=1):
```

Comment:
missing documentation, check for example the returnn jobs for correct format

Reply:
Also, if this job should only be used for a "Dynamic" concurrent value, you can remove the default "1" here and set the type to `Delayed` only.